incubator/kiam/templates/agent-daemonset.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    app: {{ template "kiam.name" . }}
+    chart: {{ template "kiam.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "kiam.fullname" . }}-agent
+  namespace: {{ .Release.Namespace }}
```

Comment:
Remove namespaces from resources. The user can provide the namespace when they install the chart.

Reply:
Fixed.